spotifython/playlist.py

```
@@ -1,228 +1,350 @@
-
-        num_tracks: An integer specifying the number of tracks to return. Can
-                    be omitted to return as many tracks as are present at
-                    start.
+        Args:
+            start: An integer specifying the 0-indexed position of the first
+                track to return. Can be omitted to return tracks starting from
+                the first song in the playlist.
+            num_tracks: An integer specifying the number of tracks to return.
+                Can be omitted to return as many tracks as are present at
+                start.
 
         Returns:
-        A Response object containing a list of Track objects.
+            A list of Track objects.
         """
-        pass
+        endpoint = Endpoints.BASE_URI
+        endpoint += Endpoints.PLAYLIST_TRACKS.replace('{playlist_id}',
+                                                      self._raw['id'])
+        if not num_tracks:
+            num_tracks = sys.maxsize
+        tracks = utils.paginate_get(self._session, num_tracks, Track,
+                                    endpoint)
+        return tracks[start:]
 
     # TODO test this in practice, what does it actually mean? Nobody knows.
-    # DELETE https://api.spotify.com/v1/playlists/{playlist_id}/tracks
-    def remove_tracks(self, tracks: Union[Track, List[Track]]=None, positions:
-                      Union[int, List[int]]=None):
-        """
-        Remove one or more tracks from the playlist.
+    def remove_tracks(self, tracks=None, positions=None):
+        """ Removes one or more tracks from the playlist.
 
         When provided with a list of tracks, all occurrences of those tracks in
         the playlist will be removed. Providing a list of track positions will
         only remove the tracks in those positions from the playlist. No tracks
         will be removed if neither of the parameters are provided. If both
         parameters are provided, positions takes precedence.
```

This sounds like too much of a transliteration of the REST api. If both (or neither) params are provided, it's probably a programmer error, and should raise an exception.